content/docs/tasks/security/auth-sds/index.md

```diff
@@ -80,3 +82,11 @@ $ kubectl delete ns foo
 $ kubectl delete ns bar
 $ kubectl delete -f istio-auth-sds.yaml
 {{< /text >}}
+
+## Caveats
+
+Currently, the SDS identity provision flow has the following caveats:
+
+* The control plane security has to be turned off. Enabling SDS with control plane security turned on is working in progress.
```

Comment:
The text fails to communicate how the one impacts the other. Make that impact explicit for clarity:
```suggestion
* To enable SDS, you must turn off the control plane security. Enabling SDS while the control plane security is on remains a work in progress.
```